src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -25,7 +25,7 @@
     @SuppressWarnings("PMD.UnusedPrivateField")
     @PrimaryKey
     @Persistent
-    private transient String feedbackSessionId;
+    private transient String feedbackSessionId; // NOPMD
```

Comment:
Just suppressing another warning (I presume this is `SingularField` violation) should work, right?